bot.py

```
@@ -0,0 +1,248 @@
+from pathlib import Path
+import random
+import re
+from typing import List, Tuple, Union
+import discord
+from discord.ext import commands
+from chatbot.chat import personal_parser, update_personal_details, check_diagnosis
+
+# from langchain_core.messages import AIMessage, HumanMessage
+from discord_bot.memory import (
```

Flake8 (7.2.0)

[error] 'discord_bot.memory.clear_user_data' imported but unused

(F401)